dashboards/redis/redis-prometheus.json

```diff
@@ -35,7 +35,7 @@
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(redis_net_output_bytes_total{${Cluster}, ${Location}, ${Namespace}} [5m])"
+                  "prometheusQuery": "rate(redis_net_input_bytes_total{${Cluster}, ${Location}, ${Namespace}} [5m])"
```

Comment:
We may want to add this to the corresponding prometheus_metadata.yaml file too as a default metric and also update the preview image.

Reply:
I added it to the metadata file. I'll see if I can find some time to stage a redis cluster to update the screenshot with actual data.